team_unified_prs.rb

```diff
@@ -39,28 +39,29 @@
     exit(3)
   end
 
-  puts "┌" + ("─" * 79)
-  puts "│   "
   all_prs = {}
 
-  team.each_with_index do |member, i|
-    next if skip_team_members.include?(member["login"])
-    prs = Github.open_pull_requests_for_involves(member["login"]).reject { |pr| pr["owner"].downcase != parsed_team["org"].downcase }
-
+  add_prs = proc do |prs|
     for pr in prs
       next if skip_pr_ids.include?(pr["url"])
       all_prs[pr["url"]] = pr
```

Comment:
I think this would be easier to understand if it were not an inline proc closure

Reply:
Where should it live then?  It references 2 local vars `skip_pr_ids` and `all_prs`.  I'm trying to avoid adding significant complexity by extracting all of those out.

Reply:
I would make the proc into a function that accepts all the required input. I think this `main` code would read easier, but this does work, so it's fine.

Reply:
I think this is cleaner imo, tho I agree that I wish ruby made lambdas/procs cleaner.  Switch to C#?